include/sys/vdev_impl.h

```diff
@@ -187,6 +199,10 @@ struct vdev {
 	boolean_t	vdev_ishole;	/* is a hole in the namespace	*/
 	kmutex_t	vdev_queue_lock; /* protects vdev_queue_depth	*/
 	uint64_t	vdev_top_zap;
+	uint64_t	vdev_alloc_bias_birth; /* allocation bias start	*/
+	uint64_t	vdev_last_io;	  /* lbolt of last non-scan I/O */
+	nvlist_t	*vdev_cfg;    /* additional dRAID configuration */
```

Comment:
I don't think this is meant to be in this PR, although I look forward to it coming soon.